.github/workflows/progress-tracker.yml

```
@@ -0,0 +1,80 @@
+name: Progress Tracker
+
+on:
+  workflow_dispatch:
+  issues:
+    types:
+      - edited
```

I'm really not a fan of running this on every issue edit. As someone who makes a lot of typos this would run it a lot. Especially as we hope this repo is about to get very active.

I think we should run on a schedule. E.g. to run at 6am and 6pm every day (UTC):

```suggestion
on:
  workflow_dispatch:
  schedule:
    - cron:  '0 6,18 * * *'
```

@rviscomi could also kick it off manual if he wants to.

---

Actually see it only does it on edits of the issue it self (not on comments below them, or even adding labels). Maybe not so bad then. But would add a few suggestions below to limit this further for edits to non-chapter issues.

---

Since we have added `workflow_dispatch` event which will allow us to run it manually, anytime we want, I think an automated scheduled update will avoid unnecessary race conditions, so let's keep it this way. If needed we can increase the frequency to every hour our something, but every 12 hours seems good to me for now.

---

I've actually come round to doing it on edits. Was worried every comment or label or raising of an issue would trigger this but it doesn't appear to so less worried than I was.

What do you think @rviscomi ?

---

+1 on edit with checks seems like a good compromise